src/axes/categoryAxis.ts

```diff
@@ -50,7 +50,7 @@ namespace Plottable.Axes {
 
     protected _setup() {
       super._setup();
-      this._measurer = new SVGTypewriter.Measurers.CacheCharacterMeasurer(this._tickLabelContainer);
+      this._measurer = new SVGTypewriter.CacheMeasurer(this._tickLabelContainer);
```

Comment:
But the cache is still useful because it can still be reused for common substrings while measuring